lib/MusicBrainz/Server/Data/LinkType.pm

```diff
@@ -94,6 +94,18 @@ around get_by_gid => sub
     return $obj;
 };
 
+sub find_by_attribute
+{
+    my ($self, $attribute_id, $limit, $offset) = @_;
+    my $query = "SELECT " . $self->_columns . "
+                 FROM " . $self->_table . "
+                     JOIN link_type_attribute_type ltat ON ltat.link_type = link_type.id
+                 WHERE ltat.attribute_type = ?
+                 ORDER BY link_type.name COLLATE musicbrainz";
+
+    $self->query_to_list_limited($query, [$attribute_id], $limit, $offset);
```

Comment:
Don't think this needs to use `query_to_list_limited`, since the number of results will always be small (and we're showing all of them in the one place that uses this).
```suggestion
    my ($self, $attribute_id) = @_;
    my $query = "SELECT " . $self->_columns . "
                 FROM " . $self->_table . "
                     JOIN link_type_attribute_type ltat ON ltat.link_type = link_type.id
                 WHERE ltat.attribute_type = ?
                 ORDER BY link_type.name COLLATE musicbrainz";

    $self->query_to_list($query, [$attribute_id]);
```